relayer/package.json

```diff
@@ -18,11 +18,12 @@
     "lint:fix": "eslint --ext .js,.ts --ignore-path .gitignore --fix .",
     "test:unit": "jest unit",
     "test:coverage": "jest --collect-coverage",
-    "generate-types": "typechain --target=ethers-v5 ./src/assets/abi/**.json --out-dir=./types-gen/contracts/",
+    "generate-types": "typechain --target=ethers-v5 ./node_modules/@beamer-bridge/deployments/dist/abis/mainnet/**.json --out-dir=./types-gen/contracts/",
     "postinstall": "yarn generate-types"
   },
   "dependencies": {
     "@arbitrum/sdk": "3.0.0",
+    "@beamer-bridge/deployments": "^2.0.1-1683711554.0",
```

Comment:
Out of curiousity: When are we planning to make a _"stable"_ release of this package?